int32 val

# overall behavior
int32 SUCCESS=1
int32 FAILURE=99999

# planning errors
int32 TRAJECTORY_CANNOT_START_WITH_WAYPOINT = -2
int32 LOCAL_TRAJECTORY_TOO_SHORT = -3

int32 UNKNOWN_STATEMENT_UID = 2
int32 UNKNOWN_POSE_UID = 3
